l10n_fi_reconcile/models/account_bank_statement.py

```diff
@@ -66,6 +66,12 @@ def _get_auto_reconcile_queries(self, params, match_amount=True):
         return sql_queries
 
     def _get_auto_reconcile_params(self):
+        # Produce parameters for SQL query
+        #
+        # Note: Odoo's float_round method can produce incorrect results
```

Comment:
Can you expand on what we mean by "incorrect results". Basically mention that the problem is that this resulting float is passed as-is into Postgresql, and so has to be exactly a particular precision, as the postgres comparison does not check precision.

Reply:
Don't be afraid to use concrete examples as necessary.

Reply:
Improved comments.